config/neo4j_data_tsv_exporter_config_example.yaml

```diff
@@ -1,14 +1,16 @@
-neo4j_user: neo4j
-neo4j_password: password
-bolt_port: "bolt://localhost:7688"
-nodes:
-  - study
-  - file
-schema:
-  - cds-model/model-desc/cds-model.yml
-  - cds-model/model-desc/cds-model-props.yml
-where:
-  study.phs_accession:
-    values:
-      - phs002790
-    where_connected: TRUE #do not need this value if the user do not want the where query affect the children nodes
\ No newline at end of file
+Config:
+  username: neo4j
+  password: password
+  bolt_port: "bolt://localhost:7687"
```

Comment:
I suggest renaming this to bolt_url or similar, bolt_port will confuse user what value is needed.

Reply:
Renaming "bolt_port" to "bolt_url" was in the latest commit.